### PR TITLE
Use bold fonts in editor

### PR DESCRIPTION
**Pull request by reduz**
* Labels are now bold
* Categories in trees are bold
* Main editor buttons are bold
* Fixed section folding arrows in inspector

image

### CODE REVIEW

**Comment by Logharaa** [comment]
I guess a before/after might have been useful to compare the changes, but seems good!

**Comment by pycbouh** [comment]
> Labels are now bold

Wait, all labels are bold by default? I'm not sure about that.

**Comment by reduz** [comment]
@pycbouh for the most part it looks good in the editor. There are so me cases where it might be good to adjust at some point, but I think its pretty good for now.

File: editor/editor_inspector.cpp [hunk 1]
```diff
@@ -1233,28 +1235,36 @@ void EditorInspectorSection::_notification(int p_what) {
...
+		c.a *= 0.4;
+		draw_rect(Rect2(Vector2(), Vector2(get_size().width, h)), c);
 
-		const int arrow_margin = 3;
-		Color color = get_theme_color("font_color", "Tree");
-		float text_width = get_size().width - Math::round((16 + arrow_margin) * EDSCALE);
-		draw_string(font, Point2(rtl ? 0 : Math::round((16 + arrow_margin) * EDSCALE), font->get_ascent(font_size) + (h - font->get_height(font_size)) / 2).floor(), label, rtl ? HALIGN_RIGHT : HALIGN_LEFT, text_width, font_size, color);
+		const int arrow_margin = 2;
+		const int arrow_width = arrow.is_valid() ? arrow->get_width() : 0;
+		//Color color = get_theme_color("font_color", "Tree");
```
Comment (Calinou) [inline editor/editor_inspector.cpp:1264]: Remove commented out code:

```suggestion
```

**Review by Calinou — DISMISSED** [review]

**Comment by Calinou** [comment]
Note that this could be remade for the `3.x` branch, since it doesn't break compatibility with existing projects. The per-TreeItem custom font feature can also be useful in other places in the editor (and possibly even in projects).

**Comment by reduz** [comment]
@Calinou to be honest I would leave 3.x alone at this point, let it conserve its visual identity so 4.0 differentiates as much as possible.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by Byteron** [comment]
In the Node section there is still no bold sections.
image

**Pull request merged**